HARK/ConsumptionSaving/ConsAggShockModel.py

```diff
@@ -1722,8 +1725,9 @@ def calcAFunc(self, MaggNow, AaggNow):
         for i in range(self.MrkvArray.shape[0]):
             these = i == MrkvHist
             slope, intercept, r_value, p_value, std_err = stats.linregress(logMagg[these], logAagg[these])
-            # if verbose:
-            #    plt.plot(logMagg[these],logAagg[these],'.')
+            if verbose:
+                #plt.plot(logMagg[these],logAagg[these],'.')
+                pass
```

Comment:
I don't understand why our core library code should have a condition that does nothingl